Source/Border.js

```diff
@@ -112,6 +112,74 @@ export class Border extends WrappedObject {
     this.sketchObject.gradientGeneric().gradientType = gradientTypeEnum;
     this.sketchApplication.reloadInspector();
   }
+  /**
+  Return a list of tests to run for this class.
+
+  @return {dictionary} A dictionary containing the tests to run. Each key is the name of a test, each value is a function which takes a Tester instance.
+  */
+
+  /**
+   Return a list of tests to run for this class.
+
+   @return {dictionary} A dictionary containing the tests to run. Each key is the name of a test, each value is a function which takes a Tester instance.
+   */
+
+  static tests() {
+    return {
+      "tests" : {
+          "testBorderThickness" : function(tester) {
+            var border = new Border()
+            border.thickness = 4;
+            tester.assertEqual(border.thickness, 4)
+          },
+
+          "testBorderPosition" : function(tester) {
+            var border = new Border()
+
+            border.positions = borderPosEnum["outside"];
+            tester.assertEqual(border.positions, 2)
+
+            border.positions = borderPosEnum["inside"];
+            tester.assertEqual(border.positions, 1)
+
+            border.positions = borderPosEnum["center"];
+            tester.assertEqual(border.positions, 0)
+          },
+
+          "testGradientTypes" : function(tester) {
+            var border = new Border()
+
+            border.gradientType = ColorHelper.gradientTypeEnum["flat"];
```

Comment:
I must admit I did agonise a little over whether to create a seperate class
for enums or just key/value pair dictionary style then. I choose the other
route given their footprint was small.

gradient is shared with borders and fills. I'm yet to implement the fills
and bring it all together ..